notesapi/v1/serializers.py

```diff
@@ -56,7 +56,7 @@ def get_text(self, note):
         Return note text.
         """
         if note.highlighted:
-            return note.highlighted[0].decode('unicode_escape')
```

Comment:
@jimabramson this `decode` method change was added in https://github.com/edx/edx-notes-api/pull/24